src/test/java/bio/terra/service/auth/iam/sam/SamIamTest.java

```diff
@@ -195,19 +196,19 @@ void testCreateDatasetResourceRequestWithPolicySpecifications() throws ApiExcept
         containsInAnyOrder(
             IamRole.ADMIN, IamRole.STEWARD, IamRole.CUSTODIAN, IamRole.SNAPSHOT_CREATOR));
 
-    AccessPolicyMembershipV2 admin = req.getPolicies().get(IamRole.ADMIN.toString());
+    AccessPolicyMembershipRequest admin = req.getPolicies().get(IamRole.ADMIN.toString());
```

Comment:
You could use `var` here to avoid work when/if the name changes again.
```suggestion
    var admin = req.getPolicies().get(IamRole.ADMIN.toString());
```

Reply:
You know it's funny, I'm still kinda torn on var.  I actually like being able to scan and view the class name

Reply:
For me it comes down to readability (which of course is subjective). I try look at the code as a series of noun/verb interactions. So it matters less what type an object is as long as its name makes sense by itself. Maybe removing the types means that we have to be better about naming our variables, which is a good thing. Of course, we also often use the Java convention of naming variables using type names so the name contains the type.

Reply:
Although I agree that it's easy to overuse `var` and we should be wary of using it all over the place.

Reply:
Totally.  I think when it's working with third party classes I'm less familiar with (like these) I like having the type up front. But like you said...personal style pref